.gitignore

```diff
@@ -13,3 +12,4 @@ tmp
 Makefile*
 mkmf.log
 Gemfile.lock
+yardoc
```

Comment:
should this be `.yardoc`?

Reply:
Both in fact! (the yardoc folder is used for generated doc in our `.yardopts` file)